src/Symfony/Component/Mailer/Bridge/Mailjet/Transport/MailjetApiTransport.php

```diff
@@ -53,11 +53,13 @@ class MailjetApiTransport extends AbstractApiTransport
 
     private string $privateKey;
     private string $publicKey;
+    private bool $sandbox;
 
-    public function __construct(string $publicKey, string $privateKey, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
+    public function __construct(string $publicKey, string $privateKey, bool $sandbox = false, HttpClientInterface $client = null, EventDispatcherInterface $dispatcher = null, LoggerInterface $logger = null)
```

Comment:
I think adding the parameter between the existing parameters might be a BC break. Because if my code was calling this constructor and passing along my own `HttpClientInterface`, it would now be passed to `$sandbox` and crash on a type error. 

Perhaps you can add the parameter as the last parameter to preserve backwards compatability?